tsconfig.json

```diff
@@ -50,7 +50,7 @@
 
     /* Emit */
     "declaration": true /* Generate .d.ts files from TypeScript and JavaScript files in your project. */,
-    "declarationMap": true /* Create sourcemaps for d.ts files. */,
+    // "declarationMap": true                            /* Create sourcemaps for d.ts files. */,
```

Comment:
Since it's unclear to me why we would remove the files, I would put them back until Marika is back, other than that LGTM

Reply:
Added it back.